libkbfs/block_types.go

```diff
@@ -188,10 +188,30 @@ func (fb *FileBlock) NewEmpty() Block {
 
 // DataVersion returns data version for this block.
 func (fb *FileBlock) DataVersion() DataVer {
+	// If this is an indirect block, and none of its children are
+	// marked as direct blocks, then this must be a big file.  Note
+	// that we do it this way, rather than returning on the first
+	// non-direct block, to support appending to existing files and
+	// making them big.
+	hasHoles := false
```

Comment:
From chat:

akalin is there a reason why `Holes` is part of `IndirectFilePtr`? i.e. why not have a `HasHoles` bool in `FileBlock`?
strib @akalin: I don't recall actually.  @taruti wrote that code, maybe she remembers?

Assuming there isn't any other reason, can you add a comment to `Holes` in `IndirectFilePtr` saying that it's there just for historical reasons, and that a `FileBlock` should be treated as having a `HasHoles` flag if any of its `IPtrs` have `Holes` set?